src/relay/backend/build_module.cc

```diff
@@ -686,9 +640,8 @@ class RelayBuildModule : public runtime::ModuleNode {
     ret_.graph_json = graph_codegen_->GetJSON();
     ret_.params = graph_codegen_->GetParams();
 
-    BuildModule(graph_codegen_->GetLoweredFunc(),
-                device_target,
-                tvm_cfg_);
+    auto target_host = Target::create(cfg_.fallback_device);
```

Comment:
Use input target_host. target_host and fallback_device is confusing, especially at the time given ```-mcpu``. We may need to only keep one?

Reply:
my bad. I forgot to change it.